getVfRep.py

```diff
@@ -0,0 +1,98 @@
+from common import TFT_TOOLS_IMG, PluginOutput, j2_render, TftAggregateOutput
+from logger import logger
+from testConfig import TestConfig
+from iperf import IperfServer, IperfClient
+from thread import ReturnValueThread
+from task import Task
+from host import Result
+from typing import Optional, Union
+import sys
+import yaml
+import json
+import jc
+
+class GetVfRep(Task):
+    def __init__(self, tft: TestConfig, iperf_instance: Union[IperfServer, IperfClient], tenant: bool):
+        super().__init__(tft, 0, iperf_instance.node_name, tenant, True)
+
+        self.in_file_template = "./manifests/tools-pod.yaml.j2"
+        self.out_file_yaml = f"./manifests/yamls/tools-pod-{self.node_name}-get-vf-rep.yaml"
```

Comment:
Same comment about the naming here, this pod is more general than just getting the vf rep

Reply:
In that case we should change up the structure of the other plugins because they use a similar naming mechanism: https://github.com/vrindle/ocp-traffic-flow-tests/blob/d611ea6823939dff8c08cf99c9f123778aa7b24f/measureCpu.py#L17C77-L17C93

Reply:
To clarify, the plugin "measureCpu" measures the CPU usage using mpstat, or the "measurePower" plugin measures the power usage etc.

This comment is just to say that we should determine a name for what this plugin does "i.e. validate-vf-rep-offload or something" and use this consistently for the task name, string provided by user to invoke the plugin, yaml files required to run the plugin etc.

Reply:
The name of this plugin implies this just gets the vf rep name. We are doing more than that